_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -331,6 +332,46 @@ $buttons-elevated-shadow: 3px !default;
 $buttons-border-radius: 3px !default;
 $buttons-font-weight: font-weight(semi-bold) !default;
 
+// forms
+$label-color: $text-base-color;
+$label-color-active: palette(grayscale, x-dark);
+
+$field-default-background: palette(grayscale, white);
+$field-default-background-focus: $field-default-background;
+$field-default-border: palette(grayscale, x-light);
+$field-default-border-focus: palette(primary, base);
+$field-default-text-color: palette(grayscale-cool, dark);
+$field-default-text-color-hover: palette(grayscale-cool, x-dark);
+
+$field-disabled-background: palette(grayscale-cool, x-light);
+$field-disabled-border: palette(grayscale-cool, light);
+$field-disabled-text-color: palette(grayscale-cool, light);
+
+$field-simple-border-focus: palette(grayscale, x-dark);
+$field-simple-background-focus: transparent;
+
+$slider-default-track: palette(grayscale, light);
+$slider-default-border: $slider-default-track;
+$slider-default-border-radius: $buttons-border-radius;
+$slider-default-handle: palette(primary, light);
+$slider-default-handle-hover: palette(primary, base);
+$slider-default-focus: palette(primary, accent);
+$slider-default-hover: palette(primary, x-accent);
+
+$slider-disabled-track: palette(grayscale, x-light);
+$slider-disabled-border: $slider-disabled-track;
+$slider-disabled-handle: palette(grayscale, base);
+$slider-disabled-focus: palette(grayscale, light);
+
+$progress-track: palette(grayscale, light);
+$progress-border: $progress-track;
+$progress-border-radius: $buttons-border-radius;
+$progress-track-done: palette(primary, accent);
+
+$helper-text: palette(grayscale-cool, dark);
```

Comment:
I don't think this variable is scoped to forms or the UI enough and the "text" suffix doesn't match how we've named other similar text color properties - I'd prefer ``$input-tip-color*`` or ``$tip-color``